CONTRIBUTING.md

```diff
@@ -69,40 +69,39 @@ or by running `pytest` inside activated environment.
 Every integration test should have `integration` mark. By default, integration tests are not execute when running `pytest`.
 
 To execute only integration tests run `hatch run integration:test` or `pytest -m integration` inside environment.
-### Connection parameters in `config.toml`
 
-Add the following connection to your `config.toml`
+### User setup
 
-```toml
-[connections.integration]
-host = <host>
-account = <account_name>
-user = <user>
-password = <password>
-```
+Integration tests require environment variables to be set up. Parameters must use the following format:
 
-### Connection parameters in environment parameters
+``SNOWFLAKE_CONNECTIONS_INTEGRATION_<key>=<value>``
 
-Parameters must use the following format:
+where ``<key>`` is the name of the key. The following environment variables are required:
 
-``SNOWFLAKE_CONNECTIONS_INTEGRATION_<key>=<value>``
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_HOST`
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT`
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_USER`
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_PASSWORD` or `SNOWFLAKE_CONNECTIONS_INTEGRATION_PRIVATE_KEY_PATH` (if using private key authentication `SNOWFLAKE_CONNECTIONS_INTEGRATION_PASSWORD=SNOWFLAKE_JWT` should be set)
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_ROLE`
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_DATABASE`
+- `SNOWFLAKE_CONNECTIONS_INTEGRATION_WAREHOUSE`
 
-where ``<key>`` is the name of the key
+### Integration account setup script
 
-For example: SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT="my-account"
+To set up an account for integration tests, run the following script with `ACCOUNTADMIN` role:
 
-List of required parameter keys:
-- host
-- account
-- user
-- password
+```bash
+snow sql -f tests_integration/scripts/integration_account_setup.sql -D "user=snowcli_test" -D "role=integration_tests" -D "warehouse=xsmall" -D "main_database=snowcli_db" -c <your_connection_name>
```

Comment:
```suggestion
snow sql \
  -f tests_integration/scripts/integration_account_setup.sql \
  -D "user=${SNOWFLAKE_CONNECTIONS_INTEGRATION_USER}" \
  -D "role=${SNOWFLAKE_CONNECTIONS_INTEGRATION_ROLE}" \
  -D "warehouse=${SNOWFLAKE_CONNECTIONS_INTEGRATION_WAREHOUSE}" \
  -D "main_database=${SNOWFLAKE_CONNECTIONS_INTEGRATION_DATABASE}"\
  -c <your_connection_name>
```

Can we use the environment variables like this?

Reply:
We can! 🚀